src/MarketPlace.sol

```diff
@@ -125,36 +128,20 @@ contract MarketPlace {
     /// @param u address of an underlying asset
     /// @param m maturity (timestamp) of the market
     /// @param t principal token addresses for this market
+    /// @param a adapter addresses for this market
     /// @param n name for the Illuminate token
     /// @param s symbol for the Illuminate token
-    /// @param a address of the APWine router that corresponds to this market
-    /// @param e address of the Element vault that corresponds to this market
-    /// @param h address of a helper contract, used for Sense approvals if active in the market
-    /// @param sensePeriphery address of the Sense periphery contract that must be approved by the lender
     /// @return bool true if successful
     function createMarket(
         address u,
         uint256 m,
-        address[8] calldata t,
+        address[] calldata t,
+        address[] calldata a,
         string calldata n,
-        string calldata s,
-        address a,
-        address e,
```

Comment:
With the removal of these params which are for verification within the lender, I'm guessing they end up on the adapters?

Reply:
Yes, we will be storing protocol specific routers within their respective adapters instead on the Lender contract itself.